tests/e2e/ctl_v3_member_test.go

```diff
@@ -55,6 +59,91 @@ func TestCtlV3MemberUpdatePeerTLS(t *testing.T) {
 	testCtl(t, memberUpdateTest, withCfg(*e2e.NewConfigPeerTLS()))
 }
 
+// TestCtlV3ConsistentMemberList requires the gofailpoint to be enabled.
+// If you execute this case locally, please do not forget to execute
+// `make gofail-enable`.
+func TestCtlV3ConsistentMemberList(t *testing.T) {
+	e2e.BeforeTest(t)
+
+	ctx := context.Background()
+
+	epc, err := e2e.NewEtcdProcessCluster(ctx, t,
+		e2e.WithClusterSize(1),
+		e2e.WithWaitClusterReadyTimeout(0),
```

Comment:
Oh, don't know how I missed this. Setting `0` here means that flag is not passed by the e2e framework. The minimum you can set is `1`, which would be `1ns`.
cc @ahrtr

Reply:
thx & fixed in https://github.com/etcd-io/etcd/pull/16671

The reason why I could still reproduce the issue is because there are 3 CONF changes, which will take 2 * 3 = 6s. The default wait-cluster-ready-timeout is 5s, so it's still easy to reproduce the issue.